packages/nimble-components/src/select/index.ts

```diff
@@ -644,8 +644,8 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
     }
 
     protected override focusAndScrollOptionIntoView(): void {
```

Comment:
Looks like mention listbox overrides the same method. Adding the same check seems reasonable there: https://github.com/ni/nimble/blame/2abc445190bb6b00dcb977b0f15a77e6d9ea3953/packages/nimble-components/src/rich-text/mention-listbox/index.ts#L257-L263

Looks like the foundation combobox overrides the same method. Seems like having the nimble combobox override the method and adding the same check before calling super would be reasonable there:
https://github.com/microsoft/fast/blob/97ef4d7a3880212bbb4f895459a047429a113472/packages/web-components/fast-foundation/src/combobox/combobox.ts#L340-L349

The check seems reasonable for all these places. Why would they try and focus something when not open